fe/fe-core/src/main/java/com/starrocks/service/arrow/flight/sql/session/ArrowFlightSqlTokenInfo.java

```diff
@@ -21,23 +21,23 @@
 
 public class ArrowFlightSqlTokenInfo {
```

Comment:
Added comments to explain what ArrowFlightSqlTokenInfo represents and how to use it